lib/yt/version.rb

```diff
@@ -1,3 +1,3 @@
 module Yt
-  VERSION = '0.33.4'
+  VERSION = '0.34.0'
```

Comment:
The readme says this gem uses semver. This could be a breaking change for anyone on Ruby < 2.1. Although the gem isn't even at 1.0, typically there's leniency with breaking changes. 

How do we feel about this? The readme is misleading at this point IMO

@justinhoward thoughts?

Reply:
Actually this gem has been increasing the second number when there's a breaking change, just the patch number increase for non-breaking changes, and never increased first number of version 😅 If you want a differently numbered version value, we can do that from now on of course.

Reply:
I get that but it's just internal knowledge of how the version bumping works instead of truly following semver which the readme describes and people may depend on.

Should we add a blurb about that in the readme?

Reply:
I wish it is ok to update CHANGELOG better, to clarify that this is a breaking upgrade.